mbq/client/contrib/permissions.py

```diff
@@ -132,6 +132,22 @@ def fetch_all_permissions(self, person_id: str) -> FetchedPermissionsDoc:
             raise ServerError("Server error") from e
 
 
+class Registrar:
+    def __init__(self):
+        self._registry: Dict[str, List[Callable[..., Any]]] = {}
+
+    def register(self, name: str, func: Callable[..., Any]) -> None:
```

Comment:
Lil' docstrings please

Reply:
Also nit: generally `fn` is used as a convention for the word function in stuff like this, since func and function are keywords in a bunch of languages (though not Python)

Whatever you do keep it consistent throughout though, you're calling it function in some places and func in others